src/components/studies/scheduler/read-only-pages/ReadOnlyNotificationWindow.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import {Box, makeStyles} from '@material-ui/core'
+import NotifcationIcon from '../../../../assets/bell.svg'
+import {latoFont, poppinsFont} from '../../../../style/theme'
+import _ from 'lodash'
+
+const useStyles = makeStyles(theme => ({
+  container: {
+    width: '100%',
+    backgroundColor: 'white',
+    marginBottom: theme.spacing(3),
+    padding: theme.spacing(2, 3),
+    fontSize: '16px',
+    fontFamily: latoFont,
+  },
+  row: {
+    display: 'flex',
+    flexDirection: 'row',
+    width: '100%',
+    justifyContent: 'space-between',
+  },
+  colContainer: {
+    flexDirection: 'column',
+    width: '100%',
+  },
+  textWithGrayBackground: {
+    width: '100%',
+    padding: theme.spacing(2, 2),
+    backgroundColor: '#F8F8F8',
+    lineHeight: '20px',
+  },
+  timeFrameText: {
+    fontFamily: poppinsFont,
+    fontSize: '18px',
+    lineHeight: '27px',
+    justifySelf: 'flex-start',
+  },
+  rowClose: {
```

Comment:
this is basically row class w/ a couple of style changes. Since its only used once - makes sense to use row and override the styles inline